front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -143,19 +127,8 @@ const AdminProjectEventEdit = () => {
   useEffect(() => {
     // Check that the event has loaded and only then can we be sure if we are creating a new one or using an existing one
     if (!isInitialLoading) {
-      const initialRoundedStartDate = roundToNearestMultipleOfFive(new Date());
-      const initialRoundedEndDate = calculateRoundedEndDate(
-        initialRoundedStartDate
-      );
-
-      setAttributeDiff({
-        start_at: event
-          ? event.data.attributes.start_at
-          : initialRoundedStartDate.toISOString(),
-        end_at: event
-          ? event.data.attributes.end_at
-          : initialRoundedEndDate.toISOString(),
-      });
+      if (event) return;
+      setAttributeDiff(initializeEventTimes());
```

Comment:
A comment would have sped up understanding of this code for me. :-)
```suggestion
      // if we're creating a new event, initialize start/end date 
      setAttributeDiff(initializeEventTimes());
```

Reply:
(Would still be helpful if we'd go for the param check as described earlier.)